compose/cli/command.py

```diff
@@ -188,3 +188,12 @@ def normalize_name(name):
         return normalize_name(project)
 
     return 'default'
+
+
+def compatibility_from_options(working_dir, options=None, environment=None):
+    """Get compose v3 compatibility options from environment then from options."""
+
+    if 'COMPOSE_COMPATIBILITY' in environment:
+        return environment.get_boolean('COMPOSE_COMPATIBILITY')
```

Comment:
In this case the environment takes the precedence that should be the opposite.
What if environment variable is set to `COMPOSE_COMPATIBILITY=no` and command line uses `--compatibility`?

Could you please, add a test using both?

Reply:
I think i have overlooked somehow the way the other functions handle precedence for options and environment variables. I will fix the precedence order :ok_hand:.

Now, if i have understood the precedence (:point_up:) correctly and we have both `COMPOSE_COMPATIBILITY=no` and `--compatibility`, `docker-compose` should results in a compatibility mode execution.
If that's the case, I can indeed add a test that cover this case.

Reply:
Yep! The command line should have precedence over the environment variable

Reply:
:ok_hand: My last push (with squashing to make commit tidier) should do the job.

To be noted: `COMPOSE_COMPATIBILITY=no` would result in a True value since `Environment.get_boolean(key)` test negative value only and handle only `false` or `0`.